dds_web/__init__.py

```diff
@@ -6,8 +6,9 @@
 
 # Standard library
 from datetime import datetime, timedelta
-import pytz
 import logging
+import pytz
+import time
```

Comment:
Are these imports here to be accessible for a separate file? It seems `time` is not used in this `__init__.py` file for example.

Reply:
`pytz` is used in `__init__.py` but `time` was probable used in a function I deleted or moved to another module, so I'll remove it 👍🏻